src/_base/helm/preview/_twig/values.yaml/environment.yml.twig

```diff
@@ -6,3 +6,5 @@
   DB_PASS: {{ @('database.pass') }}
   DB_NAME: {{ @('database.name') }}
   DB_ROOT_PASS: {{ @('database.root_pass') }}
+  DRUPAL_DOCROOT: {{ @('drupal.docroot') }}
```

Comment:
Drupal specific changes should be done in the D8 harness

Reply:
this was a leftover